### PR TITLE
[BOOKAROOM-41] Add Amenities and Swipe gesture

### PR DESCRIPTION
**Pull request by srchang8**
Added Amenities
image

Add swipe illustration
image

TODO:
Fix swipe illustration gesture with room loading time
Limit swipe on the time scroll
Change range slider color

### CODE REVIEW

File: comeet/GestureGuides/GestureGuides.swift [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//
+//  GestureGuides.swift
+//  comeet
+//
+//  Created by stephen chang on 4/25/17.
+//  Copyright © 2017 teamawesome. All rights reserved.
+//
+
+import Foundation
```
Comment (ricardocontrerasrobles) [inline comeet/GestureGuides/GestureGuides.swift]: What is this file doing? It's just importing the Foundation framework.

File: comeet/GestureGuides/GestureGuidesButton.swift [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//
+//  GestureGuidesButton.swift
+//  comeet
+//
+//  Created by stephen chang on 4/25/17.
+//  Copyright © 2017 teamawesome. All rights reserved.
+//
+
+import Foundation
```
Comment (ricardocontrerasrobles) [inline comeet/GestureGuides/GestureGuidesButton.swift]: What is this file doing? It's just importing the Foundation framework.

File: comeet/GestureGuides/GestureGuidesInterface.swift [hunk 3]
```diff
@@ -0,0 +1,9 @@
+//
+//  GestureGuidesInterface.swift
+//  comeet
+//
+//  Created by stephen chang on 4/25/17.
+//  Copyright © 2017 teamawesome. All rights reserved.
+//
+
+import Foundation
```
Comment (ricardocontrerasrobles) [inline comeet/GestureGuides/GestureGuidesInterface.swift]: What is this file doing? It's just importing the Foundation framework.

File: comeet/GestureGuides/GestureGuidesView.swift [hunk 4]
```diff
@@ -0,0 +1,9 @@
+//
+//  GestureGuidesView.swift
+//  comeet
+//
+//  Created by stephen chang on 4/25/17.
+//  Copyright © 2017 teamawesome. All rights reserved.
+//
+
+import Foundation
```
Comment (ricardocontrerasrobles) [inline comeet/GestureGuides/GestureGuidesView.swift]: What is this file doing? It's just importing the Foundation framework.

**Comment by ricardocontrerasrobles** [comment]
I see you are adding the images. But I don't see the mapping or any other code to achieve what the images in the PR description represent. Are there more commits pending? Is this work in progress?

**Comment by srchang8** [comment]
sorry, made a cp of the directory so some changes werent tracked. Fixed on the next pr, also removed the gesture files that werent being used.

**Comment by ricardocontrerasrobles** [comment]
Superseded by https://github.com/bookaroom/comeet-iOS/pull/36

**Pull request closed without merging**